library/core/src/macros/mod.rs

```diff
@@ -1295,6 +1295,14 @@ pub(crate) mod builtin {
     /// first macro invocation leading up to the invocation of the `file!`
     /// macro.
     ///
+    /// The file name is derived from the root module's source path passed to the Rust compiler
+    /// (e.g. `./src/lib.rs`)
+    /// end the sequence the compiler takes to get from root module to the module containing `file!`.
```

Comment:
```suggestion
    /// and the sequence the compiler takes to get from root module to the module containing `file!`.
```

Reply:
Also, similar to Josh's earlier note to this effect, while there's perhaps no global style for doc comments, I'd expect to see this line wrapped in a way that is similar to the existing parts of this doc comment and to the other nearby doc comments.